### PR TITLE
feat: Revise the RESTful bulk insert API

### PR DESCRIPTION
**Pull request by bigsheeper**
Revise the RESTful bulk insert API from version 1 to version 2.

issue: https://github.com/milvus-io/milvus/issues/28521

### CODE REVIEW

File: internal/distributed/proxy/httpserver/handler_v1.go [hunk 1]
```diff
@@ -946,3 +952,149 @@ func (h *HandlersV1) search(c *gin.Context) {
...
+		Files: lo.Map(httpReq.Files, func(paths []string, _ int) *internalpb.ImportFile {
+			return &internalpb.ImportFile{Paths: paths}
+		}),
+		Options:        funcutil.Map2KeyValuePair(httpReq.Options),
+		ClusteringInfo: httpReq.ClusteringInfo,
+	}
+	username, _ := c.Get(ContextUsername)
+	ctx := proxy.NewContextWithMetadata(c, username.(string), req.GetDbName())
+	response, err := h.executeRestRequestInterceptor(ctx, c, req, func(reqCtx context.Context, req any) (any, error) {
+		return h.proxy.ImportV2(ctx, req.(*internalpb.ImportRequest))
```
Comment (PowderLi) [inline internal/distributed/proxy/httpserver/handler_v1.go:989]: better to use
```
return h.proxy.ImportV2(reqCtx, req.(*internalpb.ImportRequest))
```

File: internal/distributed/proxy/httpserver/handler_v1.go [hunk 2]
```diff
@@ -946,3 +952,149 @@ func (h *HandlersV1) search(c *gin.Context) {
...
+		resp := response.(*internalpb.ImportResponse)
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusOK, HTTPReturnImportRequestID: resp.GetRequestID()})
+	}
+}
+
+func (h *HandlersV1) getImportProgress(c *gin.Context) {
+	httpReq := GetImportProgressReq{
+		DbName: DefaultDbName,
+	}
+	if err := c.ShouldBindWith(&httpReq, binding.JSON); err != nil {
```
Comment (PowderLi) [inline internal/distributed/proxy/httpserver/handler_v1.go:1009]: Http GET method, always use `c.Query($paramName, $defaultValue)
for example: $path + "? requestID=XXX&dbName=XXX

* both getImportProgress and listImports *

**Comment by samhuang-z** [comment]
rerun ut

File: internal/distributed/proxy/httpserver/handler_v2.go [hunk 3]
```diff
@@ -1536,97 +1539,76 @@ func (h *HandlersV2) alterAlias(ctx context.Context, c *gin.Context, anyReq any,
...
+		}),
+		Options: funcutil.Map2KeyValuePair(optionsGetter.GetOptions()),
 	}
 	resp, err := wrapperProxy(ctx, c, req, h.checkAuth, false, func(reqCtx context.Context, req any) (interface{}, error) {
-		return h.proxy.Import(reqCtx, req.(*milvuspb.ImportRequest))
+		return h.proxy.ImportV2(reqCtx, req.(*internalpb.ImportRequest))
 	})
 	if err == nil {
-		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusOK, HTTPReturnData: resp.(*milvuspb.ImportResponse).Tasks})
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusOK, "jobID": resp.(*internalpb.ImportResponse).GetJobID()})
```
Comment (PowderLi) [inline internal/distributed/proxy/httpserver/handler_v2.go:1589]: 保持风格统一：{code: XXX, data: YYY}
你可以用：{code: XXX, data: ${jobID}} / {code: XXX, data: {jobID: ${jobID}}}
不要改成：{code: XXX, jobID: ${jobID}}

Comment (bigsheeper) [reply internal/distributed/proxy/httpserver/handler_v2.go]: updated

**Pull request merged**